portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,14 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private int pageViews = 0;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    pageViews++;
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Zhixuan!</h1>");
+    response.getWriter().println("<p>This page has been viewed " + pageViews + " times.</p>");
```

Comment:
I added a counter to show how many times the page has been viewed.

Reply:
![image](https://user-images.githubusercontent.com/97866407/173146843-980af2db-145e-42d3-8b99-090ecea17e61.png)